src/ontology/hp-edit.owl

```diff
@@ -140931,6 +141010,555 @@ AnnotationAssertion(dcterms:date <http://purl.obolibrary.org/obo/HP_0020227> "20
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/HP_0020227> "Floating gallbladder")
 SubClassOf(<http://purl.obolibrary.org/obo/HP_0020227> <http://purl.obolibrary.org/obo/HP_0012437>)
 
+# Class: <http://purl.obolibrary.org/obo/HP_0020228> (Biospecimen phenotypic feature)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/HP_0020228> "A phenotypic abnormality obvserved in a biospecimen such as a biopsy or an extirpated tumor or tissue.")
+AnnotationAssertion(dcterms:creator <http://purl.obolibrary.org/obo/HP_0020228> <https://orcid.org/0000-0002-0736-9199>)
+AnnotationAssertion(dcterms:date <http://purl.obolibrary.org/obo/HP_0020228> "2023-11-10T16:31:31Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/HP_0020228> "Biospecimen phenotypic feature")
+SubClassOf(<http://purl.obolibrary.org/obo/HP_0020228> <http://purl.obolibrary.org/obo/HP_0000001>)
+
+# Class: <http://purl.obolibrary.org/obo/HP_0020229> (Abnormal lymph-node biospecimen)
+
+AnnotationAssertion(dcterms:creator <http://purl.obolibrary.org/obo/HP_0020229> <https://orcid.org/0000-0002-0736-9199>)
+AnnotationAssertion(dcterms:date <http://purl.obolibrary.org/obo/HP_0020229> "2023-11-10T16:45:57Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/HP_0020229> "Abnormal lymph-node biospecimen")
+SubClassOf(<http://purl.obolibrary.org/obo/HP_0020229> <http://purl.obolibrary.org/obo/HP_0020228>)
+
+# Class: <http://purl.obolibrary.org/obo/HP_0020230> (Abnormal lymph node tissue architecture)
+
+AnnotationAssertion(dcterms:creator <http://purl.obolibrary.org/obo/HP_0020230> <https://orcid.org/0000-0002-0736-9199>)
```

Comment:
A number of classes are missing definitions - I would recommend to avoid classes without definitions moving forward.